ingestion/tests/unit/helpers_test.py

```diff
@@ -68,9 +70,46 @@ def test_create_get_list_databases(self):
         updated_database = self._client.create_database(update_database_request)
         self.assertEqual(updated_database.description, created_database.description)
 
-    def test_create_update_table(self):
+    def test_5_create_update_table(self):
         databases = self._client.list_databases()
         columns = [Column(name="id", columnDataType="INT"), Column(name="name", columnDataType="VARCHAR")]
         table = CreateTableEntityRequest(name="test1", columns=columns, database=databases[0].id)
         created_table = self._client.create_or_update_table(table)
         self.assertEqual(table.name, created_table.name)
+
+    def test_6_delete_service(self):
+        mysql_service = self._client.get_database_service('local_mysql_test')
+        self._client.delete_database_service(mysql_service.id.__root__)
+        self.assertRaises(APIError, self._client.get_database_service_by_id, mysql_service.id.__root__)
+
+    def test_7_create_messaging_service(self):
+        create_messaging_service = CreateMessagingServiceEntityRequest(
+            name='sample_kafka_test',
+            serviceType='Kafka',
+            brokers=['localhost:9092'],
+            schemaRegistry='http://localhost:8081'
+        )
+        print(create_messaging_service)
```

Comment:
Can you remove the print statement over here and on other test methods as well?

Reply:
Addressed in the latest commit